src/irmin-graphql/dune

```diff
@@ -1,5 +1,5 @@
 (library
  (name irmin_graphql)
  (public_name irmin-graphql)
- (libraries fmt cohttp cohttp-lwt graphql-cohttp graphql graphql-lwt
+ (libraries mimic fmt cohttp cohttp-lwt graphql-cohttp graphql graphql-lwt
```

Comment:
same question here ; I'm not super happy to see this leaking in unrelated code.

Reply:
Seems like an bad `sed` over `conduit` and `mimic`.